apps/website/src/pages/api/cron.ts

```diff
@@ -1,10 +1,23 @@
 import type { NextApiRequest, NextApiResponse } from "next";
+
+import { env } from "@/env";
+import timingSafeCompareString from "@/server/utils/timing-safe-compare-string";
 import { runScheduledTasks } from "@/server/scheduler";
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse,
 ) {
+  // Check for secret to confirm this is a valid request
+  const authHeader = req.headers["authorization"];
+  if (
+    env.CRON_SECRET &&
+    (authHeader === undefined ||
+      !timingSafeCompareString(authHeader, `Bearer ${env.CRON_SECRET}`))
```

Comment:
```suggestion
  const { authorization } = req.headers;
  if (
    env.CRON_SECRET &&
    (authorization === undefined ||
      !timingSafeCompareString(authorization, `Bearer ${env.CRON_SECRET}`))
```
nit for consistency?